### ISSUE TITLE
Missing constructor migrations for TabParagraphContainer change

### ISSUE THREAD

**Issue by Zeyber** [SAP/spartacus#15361]
Migrations need to be added for this constructor deprecation for 5.0.
https://github.com/SAP/spartacus/pull/14880/files#r824794009
